conf/cmi/field.field.node.job_listing.body.yml

```diff
@@ -0,0 +1,23 @@
+uuid: 1bd5a81f-6ee1-46dd-8136-d22d34d59989
```

Comment:
Should this "Body" field be called more specifically "Job description"? (Just a thought tho..)
![image](https://user-images.githubusercontent.com/1712902/166200583-1df9222d-7fb9-4b2a-a899-935b677d55aa.png)